src/ucp/wireup/select.c

```diff
@@ -769,6 +798,51 @@ static ucs_status_t ucp_wireup_add_am_lane(ucp_ep_h ep, const ucp_ep_params_t *p
     return UCS_OK;
 }
 
+static ucs_status_t ucp_wireup_add_rma_bw_lanes(ucp_ep_h ep,
+                                                const ucp_ep_params_t *params,
+                                                unsigned address_count,
+                                                const ucp_address_entry_t *address_list,
+                                                ucp_wireup_lane_desc_t *lane_descs,
+                                                ucp_lane_index_t *num_lanes_p)
+{
+    ucp_wireup_criteria_t criteria;
+    ucs_status_t status;
+
+    if (ucp_ep_get_context_features(ep) & UCP_FEATURE_RMA) {
+        /* if needed for RMA, need also access for remote allocated memory */
+        criteria.remote_md_flags = criteria.local_md_flags = 0;
```

Comment:
why are flags 0?

Reply:
for RMA, we don't limit the TL criteria to access to registered memory only, we need to select both registered and allocated (0 means "any")

Reply:
this would be the same criteria as for regular RMA

Reply:
ok